benefits/eligibility/templates/eligibility/index.html

```diff
@@ -0,0 +1,25 @@
+{% extends "core/base.html" %}
+
+{% block main_content %}
+    <div class="container">
+        <div class="row justify-content-center">
+            <div class="col-lg-10">
+                {% block icon_title %}
+                    <h1>{{ page.headline }}</h1>
+                {% endblock icon_title %}
+
+                {% block paragraph_content %}
+                    {% for p in page.paragraphs %}<p style="padding-top: 24px;">{{ p }}</p>{% endfor %}
+                {% endblock paragraph_content %}
+            </div>
+        </div>
+
+        <div class="row">
+            {% block forms %}
+                {% for f in page.forms %}
+                    {% include "core/includes/form.html" with classes=page.classes form=f %}
```

Comment:
This seems strange to pass `page.classes` to the form. 

Can we edit the form class and [the view](https://github.com/cal-itp/benefits/blob/feat/947-forms--buttons-alignment-styles/benefits/eligibility/views.py#L49
), to pass the classes directly to the form?

```python
def index(request):

    # ...

     page = viewmodels.Page(
        title=_("eligibility.pages.index.title"),
        headline=_("eligibility.pages.index.headline"),
        paragraphs=[
            format_html(_("eligibility.pages.index.p[0]%(info_link)s") % {"info_link": f"{help_page}#what-is-cal-itp"})
        ],
        forms=forms.EligibilityVerifierSelectionForm(agency=agency, classes="offset-lg-1 col-lg-9"),
    )
```

Or even just hardcode them in the form class, since we always want to display this form the same.

Reply:
Tried doing this here: https://github.com/cal-itp/benefits/pull/1023/commits/38bbf9913ed615a6c6d6850ba5c4e895a77dbe85 and undoing past changes here https://github.com/cal-itp/benefits/pull/1023/commits/49690c9bd638556a1ead33f520160ab5985387e2

Reply:
Sorry to open up a resolved conversation, but couldn't we put these classes in the template itself? As in, put it directly in `eligibility/index.html`.

Reply:
@angela-tran do you mean to pass them in to the form include directly from the template? The classes are needed inside the form include itself, not in page template. 

That could work too, maybe makes more sense from the POV of keeping front-end stuff out of python.

Reply:
@thekaveman Yeah, that's more or less what I meant.

I was thinking more like we'd remove the wrapping `<div>`s from `form.html` and just have the page's template define them. The template would wrap the `{% for f in page.forms %}` loop.

Your implementation would also fit the idea.

Reply:
@angela-tran But they are needed/different per-form, and needed [deeper in the form structure](https://github.com/cal-itp/benefits/blob/feat/947-forms--buttons-alignment-styles/benefits/core/templates/core/includes/form.html#L10) than what a wrapper would provide:

```html
<!-- wrapper takes effect here -->

<form> <!-- include starts -->
<!-- this is all inside the include -->
  <div class="row form-container">
      <div class="{{ form.classes }}">
        {% for field in form %}
        <!-- ... -->
        {% endfor %}
     </div>
  </div>

  <!-- notice this row is inside the <form> but adjacent to the row/cols defined above by form classes -->
  <!-- so this can't be done as a wrapper -->

  {% if form.submit_value %}
      <div class="row d-flex flex-column-reverse justify-content-lg-end flex-lg-row pt-8">
```

Reply:
> But they are needed/different per-form

I would argue that the current designs don't require this, but... if we're saying we want to be prepared for pages with  multiple forms that all need different alignment from each other, I feel perfectly fine with your suggestion of passing the classes to the form include.

Reply:
Also note: The submit buttons on these two forms need to be in the same parent div as the form itself, or the form submit buttons won't submit. That's why the `base.html` template - with the `<form>` and the `call to action` button being in different divs - will not work for these two form pages. So the HTML form functionality requires this:

<img width="1177" alt="image" src="https://user-images.githubusercontent.com/3673236/195158205-d30bc9b1-8340-45c7-9275-33abf4a968a5.png">

The `Choose this Benefit` button *has* to be a child of the `<form>`, for the form to submit properly.

Reply:
I'm ok with moving this conversation to a separate GitHub issue so that this PR isn't bogged down since it already has a lot of commits. 

I still do think though that 
- it'd be better to pass the classes in somewhere closer to the front-end 
- the `form` attribute for the submit element (whether `input` or `button`) could help simplify a lot and help our pages follow a more unified approach.